roles/custom/matrix-client-element/defaults/main.yml

```diff
@@ -135,3 +135,21 @@ matrix_client_element_configuration_extension: "{{ matrix_client_element_configu
 # Holds the final Element configuration (a combination of the default and its extension).
 # You most likely don't need to touch this variable. Instead, see `matrix_client_element_configuration_default`.
 matrix_client_element_configuration: "{{ matrix_client_element_configuration_default | combine(matrix_client_element_configuration_extension, recursive=True) }}"
+
+# Enable Element Location sharing functionality
+# More info: https://element.io/blog/element-launches-e2ee-location-sharing/
+# How to host your own map tile server: https://matrix.org/docs/guides/map-tile-server
+matrix_client_element_location_sharing: false
```

Comment:
Maybe `matrix_client_element_location_sharing_enabled` is a better (more consistent with the rest of the playbook) name for this.

Reply:
Renamed `matrix_client_element_location_sharing` to `matrix_client_element_location_sharing_enabled`.